dbos/_client.py

```diff
@@ -243,11 +243,13 @@ def cancel_workflow(self, workflow_id: str) -> None:
     async def cancel_workflow_async(self, workflow_id: str) -> None:
         await asyncio.to_thread(self.cancel_workflow, workflow_id)
 
-    def resume_workflow(self, workflow_id: str) -> None:
+    def resume_workflow(self, workflow_id: str) -> WorkflowHandle[Any]:
         self._sys_db.resume_workflow(workflow_id)
+        return self.retrieve_workflow(workflow_id)
```

Comment:
In `fork` below we use `return WorkflowHandleClientPolling[R](forked_workflow_id, self._sys_db)`
Should we simply do the same here to reduce a roundtrip to the DB?